examples/demo/src/tailwindcssplugin.worker.js

```diff
@@ -0,0 +1,14 @@
+import lineClamp from '@tailwindcss/line-clamp';
+import typography from '@tailwindcss/typography';
+import { initialize } from 'monaco-tailwindcss/initializeTailwindcssWorker.js';
+
+initialize({
+  prepareTailwindConfig(tailwindConfig) {
```

Comment:
Can we support async functions? We can also have a look at this later.

Reply:
I'd like that, but currently the worker initialize is not async and it should be then too right?

Reply:
Let's keep stuff like that in another pr

Reply:
Indeed the worker initialization isn’t async, but all method calls that are made against it are. It’s a bit cumbersome, but all exposed methods could just await _“something”_.

Anyway, let’s delay it.

Reply:
Haha delay 😂